docs/conmon.8.md

```diff
@@ -0,0 +1,144 @@
+conmon 8 "User Commands"
+==================================================
+
+# NAME
+
+conmon - container monitor utility
+
+# SYNOPSIS
+
+conmon [options]
+
+# DESCRIPTION
+
+conmon is a command line program for monitorying and managing the lifecycle of
+Linux containers that follow the Open Container Initiative (OCI) format.
+
+# APPLICATION OPTIONS
+
+**--api-version**
+Conmon API version to use.
+
+**-b**, **--bundle**
+Location of the OCI Bundle path.
+
+**-c**, **--cid**
+Identification of Container.
+
+**--exec-attach**
+Attach to an exec session.
+
+**-e**, **--exec**
+Exec a command into a running container.
+
+**--exec-process-spec**
+Path to the process spec for execution.
+
+**--exit-command**
+Path to the program to execute when the container terminates its execution.
+
+**--exit-command-arg**
+Additional arg to pass to the exit command.  Can be specified multiple time.
+
+**--exit-delay**
+Delay before invoking the exit command (in seconds).
+
+**--exit-dir**
+Path to the directory where exit files are written.
+
+**-h**, **--help**
+Show help options.
+
+**-i**, **--stdin**
+Create !terminal stdin pipe.
+
+Create a "fake" main fd so that we can use the same epoll code in both cases.
```

Comment:
```suggestion
Create a "fake" main file descriptor so that we can use the same epoll code in both cases.
```
I'm not sure why a user would care about the epoll code or even what it is potentially.  Does this sentence even belong here?